.github/workflows/snapshot-release.yml

```diff
@@ -97,10 +106,23 @@ jobs:
       - name: Pull Request Notification
         uses: actions/github-script@v6
         env:
+          RELEASES: ${{ steps.changesets.outputs.result }}
           MESSAGE: ${{ steps.publish.outputs.result }}
         with:
           script: |
             console.log(process.env.MESSAGE);
+            const changeset = JSON.parse(process.env.RELEASES);
```

Comment:
Maybe we should put this operation in a `try/catch`, just in case? But it's more a nitpick, feel free to ignore it if it's not worth it.

Reply:
Good idea! Can't hurt.